### PR TITLE
EditorConfig/minimal reformatting

### PR DESCRIPTION
**Pull request by ekolve**
We aren't by any means finished defining or enforcing style/format rules, but I wanted to get biggest and what.I think will be the most impactful changes merged in: whitespace and brace formatting.  The style varies quite a bit all over the code base and can make it difficult to read. This at least addresses that issue.  I have also committed an .editorconfig file containing the rules which should be picked up by all the editors we use so that formatting should remain consistent.  I am going to open up separate issues for a few of the other formatting rules that were brought up since some may/may not be able to be automatically refactored.

The formatting rules were applied using [dotnet-format](https://github.com/dotnet/format)

### CODE REVIEW

File: unity/Assets/Editor/Instant Screenshot/ScreenshotTaker.cs [hunk 1]
```diff
@@ -4,213 +4,198 @@
...
+
+
+    public string ScreenShotName(int width, int height) {
+
+        string strPath = "";
+
+        strPath = string.Format("{0}/screen_{1}x{2}_{3}.png",
+                             path,
+                             width, height,
+                                       System.DateTime.Now.ToString("yyyy-MM-dd_HH-mm-ss"));
```
Comment (mattdeitke) [inline unity/Assets/Editor/Instant Screenshot/ScreenshotTaker.cs:187]: This whitespace looks off..

File: unity/Assets/Editor/Build.cs [hunk 2]
```diff
@@ -51,31 +48,28 @@ private static List<string> GetAllScenePaths()
...
         foreach (string f in files) {
             if (f.EndsWith(".unity")) {
-                Debug.Log ("Adding Scene " + f);
-				scenes.Add (f);
+                Debug.Log("Adding Scene " + f);
+                scenes.Add(f);
             }
         }
 
         //uncomment for faster builds for testing
```
Comment (mattdeitke) [inline unity/Assets/Editor/Build.cs:56]: also, is there any way to add a whitespace after each comment? So this becomes
```c#
// uncomment for faster builds for testing
```

Comment (ekolve) [reply unity/Assets/Editor/Build.cs:56]: I looked at the options and couldn't find anything.

**Review by Lucaweihs — APPROVED** [review]
I only did a brief scan of a few files but the formatting changes all looked good to me.

**Pull request merged**